packages/rn-tester/js/examples/Button/ButtonExample.js

```diff
@@ -28,93 +27,115 @@ exports.description = 'Simple React Native button component.';
 
 exports.examples = [
   {
-    title: 'Simple Button',
-    description: ('The title and onPress handler are required. It is ' +
-      'recommended to set accessibilityLabel to help make your app usable by ' +
-      'everyone.': string),
+    title: 'Button with default styling',
     render: function(): React.Node {
       return (
-        <RNTesterThemeContext.Consumer>
-          {theme => {
-            return (
-              <Button
-                onPress={() => onButtonPress('Simple')}
-                testID="simple_button"
-                color={theme.LinkColor}
-                title="Press Me"
-                accessibilityLabel="See an informative alert"
-              />
-            );
-          }}
-        </RNTesterThemeContext.Consumer>
+        <Button
+          onPress={() => onButtonPress('submitted')}
+          testID="button_default_styling"
+          title="Submit Application"
+          accessibilityLabel="Press to submit your application!"
+        />
       );
     },
   },
   {
-    title: 'Adjusted color',
-    description: ('Adjusts the color in a way that looks standard on each ' +
-      'platform. On iOS, the color prop controls the color of the text. On ' +
+    title: 'Button with color="red"',
+    description: ('Note: On iOS, the color prop controls the color of the text. On ' +
       'Android, the color adjusts the background color of the button.': string),
     render: function(): React.Node {
       return (
-        <RNTesterThemeContext.Consumer>
-          {theme => {
```

Comment:
The old examples used a theme context to support dark theme, is that not supported here?

Reply:
Yes @rickhanlonii

Reply:
You mean yes it's not supported? Can we support it?

Reply:
Sure Ricky

Reply:
Done!